test/e2e/specs/editor/blocks/classic.spec.js

```diff
@@ -39,7 +39,9 @@ test.describe( 'Classic', () => {
 		await expect.poll( editor.getEditedPostContent ).toBe( 'test' );
 	} );
 
-	test( 'should insert media, convert to blocks, and undo in one step', async ( {
+	// Broken upstream in Core.
+	// eslint-disable-next-line playwright/no-skipped-test
```

Comment:
```suggestion
	// Reinitiate once this ticket is fixed:
	// https://core.trac.wordpress.org/ticket/60666
	// eslint-disable-next-line playwright/no-skipped-test
```

Perhaps it would be better to clarify why?